lib/collections/addon/components/collections-submission/component.ts

```diff
@@ -58,6 +58,10 @@ export default class Submit extends Component {
     @tracked showResubmitModal = false;
 
     @bool('provider.reviewsWorkflow') collectionIsModerated!: boolean;
+    @computed('collectionSubmission.reviewsState')
```

Comment:
Out of curiosity, if you `@tracked collectionSubmission`, would you still need to make this a computed?

Reply:
It's a good idea to just start using tracked instead of computed properties (and there's already some tracked properties in this component too), so I will update this